platform/server-bundle/src/main/resources/webapp/WEB-INF/jsp/changePassword.jsp

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html ng-app="motech-dashboard">
+    <head>
+        <meta charset="UTF-8">
+        <title>MOTECH - Mobile Technology for Community Health</title>
+
+        ${mainHeader}
+
+    </head>
+
+    <body ng-show="ready" ng-controller="MotechMasterCtrl" class="body-down" ng-init="initChangePasswordViewData()">
```

Comment:
It's better now, but I still fail before I started to type in the second letter. I believe we have something in common.js or util.js that detects if typing is finished (uses some sort of delay). Can we use that here?